pmtiles/show.go

```diff
@@ -109,6 +111,10 @@ func Show(_ *log.Logger, output io.Writer, bucketURL string, key string, showHea
 					fmt.Println(k, "<object...>")
 				}
 			}
+
+			if strings.HasPrefix(bucketURL, "http") {
+				fmt.Println("xray: https://pmtiles.io/#url=" + url.QueryEscape(bucketURL+"/"+key))
```

Comment:
What if we use `web viewer:` instead of `xray:` to make it clearer that viewing the file requires an internet connection (the pmtiles CLI is otherwise self-contained and works offline)

Reply:
Resolved with 10479888bc41fe18a685938644b03b6c9c3dec1e